docs/documentation/Home.md

```diff
@@ -1,30 +1,30 @@
 # Flex Layout
 
-Angular Flex Layout provides a sophisticated layout API using FlexBox CSS + mediaQuery. This module provides Angular 
-developers with component layout features using a custom Layout API, mediaQuery observables, and injected DOM 
-flexbox-2016 css stylings.  
+Angular Flex Layout provides a sophisticated layout API using FlexBox CSS + mediaQuery. This module provides Angular
+developers with component layout features using a custom Layout API, mediaQuery observables, and injected DOM
+flexbox-2016 css stylings.
 
-The Layout engine intelligently automates the process of applying appropriate FlexBox CSS to browser view hierarchies. 
-This automation also addresses many of the complexities and workarounds encountered with the traditional, manual, 
-CSS-only application of Flexbox CSS. 
+The Layout engine intelligently automates the process of applying appropriate FlexBox CSS to browser view hierarchies.
+This automation also addresses many of the complexities and workarounds encountered with the traditional, manual,
+CSS-only application of Flexbox CSS.
 
 ![css3-flexbox-model](https://cloud.githubusercontent.com/assets/210413/20034148/49a4fb62-a382-11e6-9822-42b90dec69be.jpg)
 
-Angular Flex Layout is a pure-Typescript Layout engine; unlike the pure CSS-only implementations published in other 
-Flexbox libraries and the JS+CSS implementation of AngularJS Material Layouts. 
+Angular Flex Layout is a pure-Typescript Layout engine; unlike the pure CSS-only implementations published in other
+Flexbox libraries and the JS+CSS implementation of AngularJS Material Layouts.
 
-* This implementation of Angular Flex Layouts is independent of Angular Material (v1 or v2).
-* This implementation is currently only available for Angular (v4.1 and higher) applications.
+- This implementation of Angular Flex Layouts is independent of Angular Material (v1 or v2).
+- This implementation is currently only available for Angular (v4.1 and higher) applications.
 
 ### Featured Demo
 
-One of the hardest features to implement is a grid layout with specific column spans. Our online demo shows how easy 
+One of the hardest features to implement is a grid layout with specific column spans. Our online demo shows how easy
 this is!
 
 Live Demo:
 
-[![Featured](https://cloud.githubusercontent.com/assets/210413/24045163/216dae9c-0aec-11e7-8b22-616a4328eca3.jpg)](https://tburleson-layouts-demos.firebaseapp.com/#/stackoverflow)
+[![Featured](./images/gridWithColumnSpans.png)](https://stackblitz.com/fork/angular-flex-layout-home-grid-demo)
 
 Source Code:
 
-[![Feature Source](https://cloud.githubusercontent.com/assets/210413/24045169/247bad3c-0aec-11e7-827a-6ece0775ff39.jpg)](https://github.com/angular/flex-layout/blob/master/src/demo-app/app/stack-overflow/columnSpan.demo.ts#L28)
+[![Open on stackblitz](./images/gridWithColumnSpansSourceCode.png)](https://stackblitz.com/edit/angular-flex-layout-home-grid-demo?file=src/app/app.component.html)
```

Comment:
Same here re external demos

Reply:
Removed links to an external site and just left image - this image resides in the documentation/ images folder